src/components/Filter.tsx

```diff
@@ -0,0 +1,88 @@
+import FeatureLayer from '@arcgis/core/layers/FeatureLayer';
+import { Button, TextField } from '@ugrc/utah-design-system';
+import { useEffect, useRef } from 'react';
+import { useFilter } from './contexts/FilterProvider';
+import Purpose from './filters/Purpose';
+import { useMap } from './hooks';
+import { getStationQuery } from './queryHelpers';
+
+const emptyDefinition = '1=0';
+
+export default function Filter(): JSX.Element {
+  const { addLayers, mapView } = useMap();
+  const stationsLayer = useRef<FeatureLayer>();
+  const { filter } = useFilter();
+
+  useEffect(() => {
+    if (!mapView || !addLayers) {
+      return;
+    }
+
+    stationsLayer.current = new FeatureLayer({
+      url: 'https://wrimaps.utah.gov/arcgis/rest/services/Electrofishing/Public/MapServer/0',
```

Comment:
Is this ok to be static or will it need to come from an environment?

Reply:
This is a temporary use of the public service to allow me to work on the filters until we get UtahID wired up. I'll add a comment.